### PR TITLE
Add project data storage guidelines section

### PR DESCRIPTION
**Pull request by leewujung**

### CODE REVIEW

**Comment by asx-** [comment]
I see no problems with the update.

Since I'm potentially giving a demo project pitch I'm planning on making a repo that they could clone and edit, not sure how much that conflicts with them setting it up from scratch.

**Comment by leewujung** [comment]
Thanks @asx- for going over the changes.

> Since I'm potentially giving a demo project pitch I'm planning on making a repo that they could clone and edit, not sure how much that conflicts with them setting it up from scratch.

Tagging @valentina-s here since it's related to git.

File: pages/project_guidelines.md [hunk 1]
```diff
@@ -40,9 +40,21 @@ Building from what you learn at Oceanhackweek, what new approaches would you lik
...
 * Working on an efficient open source matchup tutorial for satellite orbital data and in situ point or trajectory data.
 * Developing an OSS algorithm development toolbox in Python to deal with a 400GB AMSR brightness temperature - in situ matchup database.
 * [Argovis - a visualualization and API for accessing Argo data](https://argovis.colorado.edu/ng/home)
 * [OHW18 Projects](https://oceanhackweek.github.io/projects.html)
+
+
+
+## Project Data Storage
+
+If your group need to store data for project use during Oceanhackweek, depending on the volume, below is what we we recommend:
```
Comment (emiliom) [inline pages/project_guidelines.md]: Change to:
If your group needs to store data for project use during Oceanhackweek, depending on the volume, below is what we recommend:

File: pages/project_guidelines.md [hunk 2]
```diff
@@ -40,9 +40,21 @@ Building from what you learn at Oceanhackweek, what new approaches would you lik
...
+- Public datasets like OOI, ARGO, satellite, etc. can be accessed remotely over OPeNDAP as much as feasible, or downloaded as part of the notebook script.
```
Comment (emiliom) [inline pages/project_guidelines.md]: Change to:
Whenever possible, public datasets like OOI, ARGO, satellite, etc. should be accessed remotely in the Jupyter notebook through standard protocols like OPeNDAP or other programmatic approaches.

File: pages/project_guidelines.md [hunk 3]
```diff
@@ -40,9 +40,21 @@ Building from what you learn at Oceanhackweek, what new approaches would you lik
...
+- **Data that are <1 MB**: can be included directly in the project repository.
```
Comment (emiliom) [inline pages/project_guidelines.md]: "project Github repository"

File: pages/project_guidelines.md [hunk 4]
```diff
@@ -40,9 +40,21 @@ Building from what you learn at Oceanhackweek, what new approaches would you lik
...
+- **Data that are <10 GB**: can be stored in your home space on ocean.pangeo.io. It is often useful to have a cell in the notebook that downloads such data into your home space if those data are not there, otherwise do nothing if they are already there. That way you can share code and notebooks over GitHub with group members, but have a way to fetch and store larger datasets locally if they are not enormous.
+- **Data that are >10 GB**: they are best stored in a cloud-optimized format. If you don't already have the data on the cloud, please talk to the instructors about your needs and how you anticipate using the data. We can help you prepare and upload the data to cloud object storage with Oceanhackweek support.
```
Comment (emiliom) [inline pages/project_guidelines.md]: **Data that are >10 GB**: are best stored in a cloud-optimized format. If you don't already have the data on the cloud, please talk to the instructors about your needs and how you anticipate using the data. We can help you prepare and upload the data to [cloud object storage](https://en.wikipedia.org/wiki/Cloud_storage) with Oceanhackweek support.

**Comment by emiliom** [comment]
I made small corrections and minor improvements to the data storage text.

**Comment by leewujung** [comment]
Thanks @emiliom! All changed. I'm merging this now.

**Pull request merged**